### ISSUE TITLE
Unit switch mg/dl mmol/L

### ISSUE THREAD

**Issue by MiGerth** [t1dnz/CamAPSSidekick#3]
Dear Graham, 

thanks a lot for this nice app. It's so helpful to know the actual IOB inclusiv it's basal. Unfortunatelyn I cant switch the bg reading from mmol/L to mg/dl. 

Perhaps you have an idea to fix it or what I've to change. 

Regards,
Michael

**Comment by PieterDeBruijn** [comment]
"Unfortunatelyn I cant switch the bg reading from mmol/L to mg/dl."
This was a feature on the roadmap to v0.1.0. So it's on the TODO list.

IOB including basal is indeed nice to have! Good suggestion.

**Comment by grahamjenson** [comment]
Hey,

Sorry for the lack of reply. I have been off of programming for the last couple months, and am just getting back into the swing of it.

I am moving all the diabetes related calculations and logic to https://github.com/t1dnz/t1djava so that it is easier to test seperate from the UI.

This will also hopefully make it much easier to implement the unit switch. My main goal at the moment is just to test all the calculations as I am not 100% they are correct. Once that work is complete I will work on more features. I don't want to promise too much, but I hope to get a lot of this done between now and April.
